ipatests/test_webui/test_group.py

```diff
@@ -416,3 +416,52 @@ def test_member_manager_group(self):
 
         self.delete(user.ENTITY, [user.DATA_MEMBER_MANAGER, user.DATA])
         self.delete(group.ENTITY, [group.DATA2, group.DATA3])
+
+    @screenshot
+    def test_settings_new_columns(self):
+        '''
+        Related: https://github.com/freeipa/freeipa/pull/6531
+        Test that the new columns 'givenname', 'sn' and 'nsaccountlock'
+        are visible when creating new user groups and inspecting its
+        'settings' section.
+        '''
+        self.init_app()
+
+        # Create new user (disabled)
+        self.add_record(user.ENTITY, user.DATA, navigate=True)
+        self.disable_action()
+
+        # Create a new user (enabled)
+        self.add_record(user.ENTITY, user.DATA2, navigate=False)
+
+        # Create an empty group
+        self.add_record(group.ENTITY, [group.DATA])
+
+        # Add created users to the group
+        self.add_associations([user.DATA, user.DATA2], facet='member_user')
+
+        # Check if new columns are in the group 'settings' section
+        # - Disabled user
+        self.navigate_to_record(group.PKEY)
+        d_first_name_column = self.find('givenname', by=self.By.NAME)
+        d_last_name_column = self.find('sn', by=self.By.NAME)
+        d_status_column = self.find('nsaccountlock', by=self.By.NAME)
+        assert d_first_name_column
+        assert d_last_name_column
+        assert d_status_column
+        # - Enabled user
+        self.navigate_to_record(group.PKEY2)
+        e_first_name_column = self.find('givenname', by=self.By.NAME)
```

Comment:
remove `self.` from `self.By.NAME`. By is not an attribute of UI driver. Rather import it to the `test_group.py` file using 

```
try:
    from selenium.webdriver.common.by import By
except ImportError:
    NO_SELENIUM = True
```
(utilize existing try-except block on line 34)